crates/core/agent_interface/Cargo.toml

```diff
@@ -12,6 +12,7 @@ description = "agent_interface defines all the mqtt topics and messages to be us
 [dependencies]
 csv = "1.1"
 download = { path = "../../common/download" }
+mqtt_channel = { path = "../../common/mqtt_channel" }
```

Comment:
Writing this I asked myself the same question. I decided for it, because this `agent_interface` exposes too many topics and messages using an un-typed manner, delegating to the mappers and the agent parsing concerns.

Using a feature to sherry-pick only `Message` and `Topic` from `mqtt_channel` is an alternative to consider.